qctrlopencontrols/dynamic_decoupling_sequences/predefined.py

```diff
@@ -126,40 +127,11 @@ def _add_pre_post_rotations(
     # of Y and Z pi-pulses
     preserves_11 = (y_pi_pulses + z_pi_pulses) % 2 == 0
 
-    # When states |0> and |0>+|1> are preserved, the sequence already produces
-    # an identity, so that we want the the pi/2-pulses to cancel each other out
-    if preserves_10 and preserves_11:
-        rabi_value = np.pi / 2
-        initial_azimuthal = 0
+    # use pi/2 and -pi/2 (pi/2) X pulses as pre and post rotations for CP sequences with
```

Comment:
I'd suggest updating this comment to be more general, since it's meant to work for more than just CP/CPMG sequences. IMO what's important is that for an identity or Y rotation we use a -pi/2 final rotation, to ensure that the total rotation is either I or Z.

Reply:
done.